content/docs/guide.md

```diff
@@ -27,14 +27,20 @@ the logic is baked inside the `packit` command - so once `packit srpm` (the comm
 to create a source RPM locally from the current snapshot of the project) starts
 passing for you, it should work inside packit service as well.
 
-This is how you can install packit if you are on Fedora Linux:
+### Fedora Linux
 
     $ sudo dnf install packit
 
-We publish packit to PyPI and it's available as `packitos` project — **packit
-at PyPI is something different**.
+### Red Hat Enterprise Linux or CentOS Stream 8
 
-    $ pip3 install --user packitos
+On RHEL/CentOS 8 you can install RPM from [EPEL](https://fedoraproject.org/wiki/EPEL) repository.
+On CentOS, some dependencies are in PowerTools repository, so you have to enable it:
+
+    $ sudo dnf install dnf-plugins-core epel-release
+    $ sudo dnf config-manager --set-enabled powertools
+    $ sudo dnf install packit
```

Comment:
This is also a better way of doing it in comparison to my PR since the powertools repo will be enabled persistently.